system/security/shared/caching.hpp

```diff
@@ -141,6 +144,7 @@ class CPermissionsCache : public CInterface
         m_lastManagedFileScopesRefresh = 0;
         m_defaultPermission = SecAccess_Unknown;
         m_secMgrClass.set(_secMgrClass);
+        m_transactionalCacheTimeout = DEFAULT_CACHE_TIMEOUT_SECONDS;
```

Comment:
It should be deleted, along with a bit more transaction stuff in caching.cpp. Will do that with separate Jira